kmir/src/kmir/kast.py

```diff
@@ -21,7 +22,7 @@ def int_var(var: KVariable, num_bytes: int, signed: bool) -> tuple[KInner, Itera
     bit_width = num_bytes * 8
     var_max = ((1 << (bit_width - 1)) if signed else (1 << bit_width)) - 1
     var_min = -(1 << (bit_width - 1)) if signed else 0
-    constraints = (leInt(var, token(var_max)), leInt(token(var_min), var))
+    constraints = (mlEqualsTrue(leInt(var, token(var_max))), mlEqualsTrue(leInt(token(var_min), var)))
     term = KApply('Value::Integer', (var, token(bit_width), token(signed)))
     return term, constraints
```

Comment:
Hmmmm, why didn't CI catch this error? Do we have any tests exercising these cases?

Reply:
I don't think any of our prove_rs integration tests are exercising functions with parameters. I'll add some that do.

Reply:
Correction: we DO have tests that work with parameterized functions.

Before fixing the `start_symbol` issue in my commit, `main` was just being called in all of the prove-rs tests, so the `mlEqualsTrue` issue was hidden.

All should be good.